### PR TITLE
feat: Added authentication pages

### PR DESCRIPTION
**Pull request by mnath30**
Added authentication pages:
1. Login page
image
2. Signup page
image
3. Reset password page
image

### CODE REVIEW

**Review by AmishMishra11 — APPROVED** [review]
Good to merge

File: pages/Authentication/login.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    <link rel="stylesheet" href="authentication.css">
+    <link rel="icon" href="../../resource/asset/thumbnail.png">
+    <title>Login-Closet</title>
+</head>
+<body>
+    <!-- Navigation -->
+    <header>
+        <nav class="navigation">
+            <div class="nav-brand-logo">
+                <img class="nav-img" src="../../resource/asset/_Logo.png">
```
Comment (devananth) [inline pages/Authentication/login.html]: `alt` attribute is missing in `img` tag

File: pages/Authentication/logout.html [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    <link rel="stylesheet" href="authentication.css">
+    <link rel="icon" href="../../resource/asset/thumbnail.png">
+    <title>Login-Closet</title>
+</head>
+<body>
+    <!-- Navigation -->
+    <header>
+        <nav class="navigation">
+            <div class="nav-brand-logo">
+                <img class="nav-img" src="../../resource/asset/_Logo.png">
```
Comment (devananth) [inline pages/Authentication/logout.html]: Fix this too

**Review by devananth — CHANGES_REQUESTED** [review]
Code seems clean. Good work ! 

Suggested one change , try doing it.

File: pages/Authentication/logout.html (hunk 2 above)
Comment (mnath30) [reply pages/Authentication/logout.html]: Thankyou for your comment. I have made the changes

**Pull request merged**